src/Glpi/Form/AnswersSet.php

```diff
@@ -265,6 +268,26 @@ public function setSubmittedFiles(array $files): void
         $this->files = $files;
     }
 
+    /**
+     * Get delegation data
+     *
+     * @return array<string, ?string>
```

Comment:
We could use a real object to store the data instead of an array with specific keys that need to be known it interact with it.